unit/util/bitvector_expr.cpp

```diff
@@ -64,3 +71,34 @@ TEMPLATE_TEST_CASE(
     }
   }
 }
+
+TEST_CASE("onehot expression lowering", "[core][util][expr]")
```

Comment:
Thank you for adding this test; but could `onehot0` please also be included?

Reply:
Done